examples/solvers/find_optimal_parameters.py

```diff
@@ -0,0 +1,234 @@
+"""
+Example of how to use optimization in order to pick reconstruction parameters.
+
+In this example, we solve the tomographic inversion problem with different
+regularizers (fbp, huber and tv) and pick the "best" regularization parameter
+for each method w.r.t. a set of reference data.
+
+To find the "best" parameter we use Powells method to optimize a figure of
+merit, here the L2-distance to the true result.
+"""
+
+import numpy as np
+import odl
+import scipy
+
+
+def optimal_parameters(reconstruction, fom, phantoms, data,
+                       initial_param=0):
+    """Find the optimal parameters for a reconstruction method.
+
+    Notes
+    -----
+    For a forward operator :math:`A : X \to Y`, a reconstruction operator
+     parametrized by :math:`\theta` is some operator :math:`R_\theta : Y \to X`
+     such that
+
+    .. math::
+        R_\theta(A(x)) \approx x.
+
+    The optimal choice of :math:`\theta` is given by
+
+    .. math::
+        \theta = \argmin_\theta fom(R(A(x) + noise), x)
```

Comment:
I suspect that some of these formulas don't render properly, at least not in Spyder. That's because they contain escape sequences like `\t` that are interpreted. Those need to be double-escaped, like `\\t`. Further candidates are `\n`, `\l`, `\r`. You can check in Spyder by putting the cursor over a function definition and hitting Ctrl+I (capital "i"), and the formulas should be rendered correctly.